modules/auth/cms_authentication_module.e

```diff
@@ -458,32 +607,22 @@ feature -- Handler
 					l_string.append (ic.item)
 					l_string.append_character (' ')
 				end
-				write_debug_log (generator + ".block_list:" + l_string )
+				write_debug_log (generator + ".block_list:" + l_string)
 			end
 		end
 
 	get_block_view (a_block_id: READABLE_STRING_8; a_response: CMS_RESPONSE)
 		do
-			if
-				a_block_id.is_case_insensitive_equal_general ("register") and then
-				a_response.location.starts_with ("account/roc-register")
-			then
+			if a_block_id.is_case_insensitive_equal_general ("register") and then a_response.location.starts_with ("account/roc-register") then
```

Comment:
Maybe it would be good timing to move from this block solution, to real page handler .. 
since registration should be in a specific page, and not in any block 

Do you see what I mean ?